### DISCUSSION TITLE
broken auto-import paths in VSCode w/ locally linked packages

### DISCUSSION THREAD

**Discussion by diminutivesloop** [yarnpkg/berry#4796] [Q&A]
My team is running into a weird issues w/ auto-imports in VSCode from locally linked packages where the slash between package name and the module path is missing. So instead of getting `import { someFunction } from '@scope/package/someFunction` we get `import { someFunction } from '@scope/packagesomeFunction`. For context we're using Yarn in PnP mode and we don't see this issue when these same packages are installed regularly instead of linked. Has anyone else run into this and figured out how to fix it?

**Comment by merceyz** [comment]
Please open a bug report instead, we need more information / a reproduction to help here.

**Reply by diminutivesloop** [reply]
So I can't reproduce this anymore which makes this whole issue moot, but I originally posted it here instead of creating an issue because I didn't have time/wasn't sure how to create a full repro and I wanted to see if other users had encountered a similar issue and found a resolution. Apologies if I misunderstood what Discussions are intended for here.
